boilerplate/app/models/EpisodeStore.ts

```diff
@@ -18,6 +19,24 @@ export const EpisodeStoreModel = types
         console.tron.error(`Error fetching episodes: ${JSON.stringify(response)}`, [])
       }
     },
+    addFavorite(episode: Episode) {
+      store.favorites.set(episode.guid, clone(episode))
```

Comment:
Rather than cloning the episode, I think we'd just want to push the GUID into the references array.

```suggestion
      store.favorites.push(episode.guid)
```

Reply:
For some reason, I didn't think references would work since we replace the whole list on refresh. But it seems to work just fine. Updated.